index.yaml

```diff
@@ -15,7 +15,7 @@ entries:
     sources:
     - https://github.com/elastic/elasticsearch
     urls:
-    - charts/elasticsearch-7.6.2.tgz
+    - legacy-charts/elasticsearch-7.6.2.tgz
```

Comment:
if you change the url, people are forced to update the index to download the chart, which can cause disruption

Reply:
why do you need a legacy-chart directory? can't you just use `charts/`? the new charts published automatically should target never versions

Reply:
Unfortunately it is necessary, as the chart releaser action is expecting the charts to be in the `charts` directory and wasn't playing nice when I changed the expected directory.